tests/quickcheck.rs

```diff
@@ -0,0 +1,326 @@
+extern crate num_bigint;
+extern crate num_integer;
+extern crate num_traits;
+
+#[cfg(test)]
+#[cfg(feature = "quickcheck")]
+#[macro_use]
+pub extern crate quickcheck;
+
+#[cfg(test)]
+#[cfg(feature = "quickcheck")]
+mod quickchecks {
+    use num_bigint::{BigInt, BigUint};
+    use num_traits::{Num, One, Pow, Zero};
+    use quickcheck::TestResult;
+
+    quickcheck! {
```

Comment:
This macro accepts repetitions, so you could write every `fn` within a single macro instance.

Reply:
Oh, I didn't realize. The way this is written is a holdover from the way I started (and the way I have previously used quickcheck) which is to have a function attribute i.e. `#[quickcheck]`. I switched from attributes to the macro in this crate to improve compatibility with old rust, but quickcheck didn't maintain compatibility with old Rust (the version requirements of the attribute are even tighter than that of the rest of quickcheck).

I prefer the quickcheck attribute on aesthetic grounds (less indentation), so I think I'll switch to that.